tests/integration/conftest.py

```diff
@@ -113,30 +112,40 @@ def _env_phoenix_sql_database_url(
     _fake: Faker,
 ) -> Iterator[None]:
     values = [(ENV_PHOENIX_SQL_DATABASE_URL, _sql_database_url.render_as_string())]
+    with mock.patch.dict(os.environ, values):
```

Comment:
How about `monkeypatch.setenv`?

Reply:
`monkeypatch` is function scope, so it can't be used in a module scope fixture